tests/migration/migration.go

```diff
@@ -906,47 +906,6 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 				// check VMI, confirm migration state
 				libmigration.ConfirmVMIPostMigration(virtClient, vmi, migration)
 			})
-
-			//TODO Could this be unit test?
-			It("[test_id:6974]should reject additional migrations on the same VMI if the first one is not finished", func() {
-				vmi := libvmifact.NewFedora(libnet.WithMasqueradeNetworking(), libvmi.WithResourceMemory(fedoraVMSize))
-
-				By("Starting the VirtualMachineInstance")
-				vmi = libvmops.RunVMIAndExpectLaunch(vmi, 240)
-
-				// Need to wait for cloud init to finish and start the agent inside the vmi.
-				Eventually(matcher.ThisVMI(vmi), 12*time.Minute, 2*time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstanceAgentConnected))
-
-				By("Checking that the VirtualMachineInstance console has expected output")
-				Expect(console.LoginToFedora(vmi)).To(Succeed())
-
-				// Only stressing the VMI for 60 seconds to ensure the first migration eventually succeeds
-				By("Stressing the VMI")
-				runStressTest(vmi, stressDefaultVMSize, 60)
-
-				By("Starting a first migration")
-				migration1 := libmigration.New(vmi.Name, vmi.Namespace)
-				migration1, err := virtClient.VirtualMachineInstanceMigration(migration1.Namespace).Create(context.Background(), migration1, metav1.CreateOptions{})
-				Expect(err).ToNot(HaveOccurred())
-
-				// Successfully tested with 40, but requests start getting throttled above 10, which is better to avoid to prevent flakyness
-				By("Starting 10 more migrations expecting all to fail to create")
-				var wg sync.WaitGroup
-				for n := 0; n < 10; n++ {
-					wg.Add(1)
-					go func(n int) {
-						defer GinkgoRecover()
-						defer wg.Done()
-						migration := libmigration.New(vmi.Name, vmi.Namespace)
-						_, err = virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(context.Background(), migration, metav1.CreateOptions{})
-						Expect(err).To(HaveOccurred(), fmt.Sprintf("Extra migration %d should have failed to create", n))
-						Expect(err.Error()).To(ContainSubstring(`admission webhook "migration-create-validator.kubevirt.io" denied the request: in-flight migration detected.`))
-					}(n)
-				}
```

Comment:
I am not convinced here. Can we leave it for follow up?

Reply:
Done!Restored